.python-version

```diff
@@ -1 +1 @@
-3.12
+3.13
```

Comment:
Note: this change will cause `uv` to recreate the venv (and automatically download Python if necessary).